### DISCUSSION TITLE
Multi-language support

### DISCUSSION THREAD

**Discussion by njlbenn** [ZUGFeRD/mustangproject#570] [General]
Hi, I am encountering and using Mustang for the first time. I don't know if this is the best place to ask this kind of question. (Apologies if it is not, and in that case could someone direct me to the right forum?)

I wanted to find out if there is a way to configure Mustang to produce error messages in other languages than English?

**Comment by jstaerk** [comment]
Not yet, nobody has started translating it. We have invoice output visualization in DE or FR but if taken seriously one would probabaly also want to translate the "error messages" of the validation of which there are a few.

**Comment by larsboehnke** [comment]
I will translate the error messages in the upcoming days
